### PR TITLE
Updating font weights, new launch screen, unifying footer and header margins

### PR DESCRIPTION
**Pull request by markesaaremets**
- New launcscreen image
- Unifying footer and header margins to be 16pt from the edges (as Conversation)
- Medium font changed to Semibold to increase the contrast beween different font weights

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/UserProfile/Views/ProfileHeaderView.swift [hunk 1]
```diff
@@ -82,7 +82,8 @@ final class ProfileHeaderView: UIView {
         }
 
         constrain(self, dismissButton, verifiedImageView, detailView.titleLabel) { (view: LayoutProxy, dismiss: LayoutProxy, verified: LayoutProxy, title: LayoutProxy) -> () in
-            dismiss.top == view.top + 24
+            dismiss.top == view.top + 26
+            dismiss.right == view.right - 8
```
Comment (daehn) [inline Wire-iOS/Sources/UserInterface/UserProfile/Views/ProfileHeaderView.swift]: The `dismiss.right` constraint might break the view in some contexts. Depending on the `headerStyle` the dismiss button is displayed on the left or the right side.

Comment (daehn) [inline Wire-iOS/Sources/UserInterface/UserProfile/Views/ProfileHeaderView.swift]: @markesaaremets see https://github.com/wireapp/wire-ios/pull/845/files#diff-1b8505afca416dff9fcb34ed891d1201R97

Comment (daehn) [inline Wire-iOS/Sources/UserInterface/UserProfile/Views/ProfileHeaderView.swift]: It looks like you can modify the `horizontalMargin` instead (`profile_temp.content_left_margin`).

Comment (jenskasemets) [inline Wire-iOS/Sources/UserInterface/UserProfile/Views/ProfileHeaderView.swift]: @daehn would it make sense to switch to `leading-trailing` here?

**Comment by daehn** [comment]
@markesaaremets The newly recorded call system message has a different timestamp? The time changed one hour 🤔

File: Wire-iOS/Sources/UserInterface/UserProfile/Views/ProfileHeaderView.swift (hunk 1 above)
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/UserProfile/Views/ProfileHeaderView.swift]: @jenskasemets may be yes, but Marke would not do it. Not like he cannot do it, but would take too much time to dig in this old ObjC code.

Comment (daehn) [inline Wire-iOS/Sources/UserInterface/UserProfile/Views/ProfileHeaderView.swift]: I think updating the magic constant is enough, as the constraint is already created here: https://github.com/wireapp/wire-ios/pull/845/files#diff-1b8505afca416dff9fcb34ed891d1201R97

**Pull request merged**